Translations-Web/lang_cs.php

```diff
@@ -0,0 +1,115 @@
+<?php // Keka lang - version 0.3 - Started on 26/11/09 - by aone
+
+//
+//
+// Language: English
+$content_language_locale = "cs";
+// Translated by aone (erben.fr@gmail.com)
+//
+//
+// Set $show_content_bottom_translator as true if you want to appear in the bottom of the page
+$show_content_bottom_translator = false;
```

Comment:
@ferben note you'll not be displayed in the bottom of the website with this set to false, just in case you missed it.

Reply:
It's OK.
I'll fix it anytime in the future, with any new localisation fixies

Reply:
Perfect. The website is already running your translation. Thanks a lot @ferben 🙌🏼

Reply:
https://www.keka.io/cs/